packages/forklift-console-plugin/src/modules/Plans/views/details/components/PlanPageHeadings.tsx

```diff
@@ -158,8 +165,37 @@ export const PlanPageHeadings: React.FC<{ name: string; namespace: string }> = (
     }
   };
 
+  const isWarmAndExecuting = plan?.spec?.warm && isPlanExecuting(plan) && !isPlanArchived(plan);
+
+  const onClickPlanCutoverMigration = () => {
+    showModal(<PlanCutoverMigrationModal resource={plan} />);
+  };
+  const cutoverTime = DateTime.fromISO(lastMigration?.spec?.cutover).toLocaleString(
+    DateTime.DATETIME_FULL,
+  );
+  const cutoverSet = Boolean(lastMigration?.spec?.cutover);
+
   const actions = (
     <Level hasGutter>
+      {isWarmAndExecuting && migrationLoaded && (
+        <>
+          {cutoverSet ? (
+            <Tooltip content={cutoverTime}>
+              <Button
+                variant="secondary"
+                icon={<CalendarIcon />}
+                onClick={onClickPlanCutoverMigration}
+              >
+                {t('Edit cutover')}
+              </Button>
+            </Tooltip>
+          ) : (
+            <Button variant="primary" icon={<CalendarIcon />} onClick={onClickPlanCutoverMigration}>
+              {t('Schedule cutover')}
+            </Button>
```

Comment:
this looks very similar to the block you made in MigrationTypeCell.tsx file,
Please extract this component and use it in both places